### DISCUSSION TITLE
Onboarding isn't getting retriggered for my repository

### DISCUSSION THREAD

**Discussion by hwittenborn** [renovatebot/renovate#22530] [Request Help]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe the problem

I recently nuked the Renovate config from my repository (makedeb/prebuilt-mpr) and renamed the onboarding PR, but the new onboarding PR hasn't come in yet. I came to make the issue here from following the instructions at https://docs.renovatebot.com/getting-started/installing-onboarding/#nuke-config-and-re-onboard.

### Have you created a minimal reproduction repository?

I have linked to a minimal reproduction in the description above

**Comment by rarkins** [comment]
From the last run 9 hours ago, Renovate thought that your repo has been _renamed_ and stopped because of that. Has any renaming happened?
